content/docs/gke/gcp-e2e.md

```diff
@@ -294,38 +294,13 @@ gsutil mb -c regional -l us-central1 gs://${BUCKET_NAME}
 The sample you downloaded contains all the code you need. If you like, you
 can experiment with and test the code in a Jupyter notebook.
 
-The Kubeflow deployment includes [JupyterHub][jupyterhub] and a
-corresponding load balancer service. You can choose to connect to JupyterHub
-using the Kubeflow URL or locally.
+The Kubeflow deployment includes services for spawning and managing 
+[Jupyter notebooks][jupyter-notebook]. 
```

Comment:
Same comment as in other locations about capitalizing Jupyter Notebook in this case.

Reply:
I'm going to stick with lower case for easier reading and for consistency across the docs, though I do take your point.